Makefile.am

```diff
@@ -50,3 +50,11 @@ dist_noinst_DATA = \
 	AUTHORS COPYING README \
 	example.c \
 	testinput.c
+
+if COMPILE_EXAMPLE
+check_PROGRAMS = example
```

Comment:
"check" is for unit tests specifically, which example (unfortunately) doesn't do today.